src/Entity/IdeasWorkshop/Vote.php

```diff
@@ -67,6 +94,7 @@ public function getIdea(): Idea
     public function setIdea(Idea $idea): void
     {
         $this->idea = $idea;
+        $this->idea->incrementVotesCount(1);
```

Comment:
This default argument value is useless here

 ```php
public function incrementVotesCount(int $increment = 1): void
```

Reply:
can you move this call to Idea class pls? in `addVote` method

Reply:
Changed. Now a listener does incrementation.